### DISCUSSION TITLE
Is MFA available?

### DISCUSSION THREAD

**Discussion by michaelhays** [vercel/community#4866] [Feature Question] [answered]
### Question

Is Multi-Factor Authentication available for Vercel accounts? I couldn't find anywhere to enable that in the Vercel dashboard.

I'm working on a security review in preparation for a SOC 2 audit, and it would be helpful to provide evidence of this.

I see in the [Vercel security page](https://security.vercel.com/?itemUid=98c0e7d7-4916-4bdc-8f2a-519ef2a315f6&source=search) that "Multi-Factor Authentication" is listed, but I'm not sure if that implies the availability for Vercel user accounts.

**Comment by ben-bcn** [comment]
I am search for the same information for the same reason.

**Comment by amyegan** [comment, marked as answer]
You can use 2FA for your Git login connection ([GitHub](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication), [GitLab](https://docs.gitlab.com/ee/user/profile/account/two_factor_authentication.html), or [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/enable-two-step-verification/)). Logging in via email requires verification through your email as well, so using 2FA with your email login can add another layer of protection.

For Enterprise customers, we also have a sign-in via SSO feature available: [vercel.com/docs/security/saml#authenticating-with-saml-sso](https://vercel.com/docs/security/saml#authenticating-with-saml-sso)

**Reply by michaelhays** [reply]
Thanks for the update! Yep that's what I'm currently doing -- 2FA on both GitHub and email.

MFA would still be much appreciated, if that's anywhere on the roadmap :)

**Reply by colincclark** [reply]
Seems like a reasonable request from quite a while ago. Why does Vercel not support MFA itself?

**Reply by swarnava** [reply]
We use Passwordless login. If you are login using your mailbox or Git provider, it should already contain MFA enabled.

**Reply by leerob** [reply]
We also support logging in with passkey! https://vercel.com/changelog/login-with-passkey-is-now-supported

[a comment by an automated account is not shown]
